runner/app/pipelines/audio_to_text.py

```diff
@@ -92,12 +99,10 @@ def __init__(self, model_id: str):
             low_cpu_mem_usage=True,
             use_safetensors=True,
             cache_dir=get_model_dir(),
+            attn_implementation=attn_implementation,
+            device_map={"": torch_device},
```

Comment:
@RUFFY-369, @eliteprox This updated code ensures correct device placement while eliminating the need to call `.to` multiple times.

Reply:
Applied https://github.com/livepeer/ai-worker/pull/308/commits/f05dfce45ca4d750ca6162f12636188713d41477.